Subtester/ProjectSettings/ProjectSettings.asset

```diff
@@ -161,7 +161,7 @@ PlayerSettings:
     Standalone: 0
     iPhone: 0
     tvOS: 0
-  overrideDefaultApplicationIdentifier: 0
+  overrideDefaultApplicationIdentifier: 1
```

Comment:
For some reason, Unity constantly changed the bundle id for me without this...

Reply:
oh yes I've seen that as well, nice